index.html

```diff
@@ -0,0 +1,99 @@
+</head>
+
+<body>
+    <header>
+        <h1>Evgeni Borovik</h1>
+        <hr>
+        <div class="photo">
+            <img src=".\assets\img\photo.jpg" alt="Evgeni Borovik photo">
+        </div>
+    </header>
+    <section class="contacts">
+        <h2 class="header-contacts">Contact information</h2>
+        <p class="paragraph"><em>email:</em> <a href="mailto:anyuser@mail.ru">anyuser@mail.ru</a>
+            <br>
+            <em>skype:</em> eugene_borovik
+            <br>
+            <em>mobile / telegram / viber:</em> <a href="tel:+375296829132">+375 29 682-91-32</a>
+        </p>
+
+    </section>
+    <hr>
+    <section>
+        <h2>Summary</h2>
+        <p class="paragraph">Current position: Strategic Planning Director
+            <br>
+            Target position: Project Manager in a software development company
+        </p>
+    </section>
+    <hr>
+    <section>
+        <h2>Skills</h2>
+        <ul>
+            <li>Project management</li>
+            <li>HTML/CSS basics</li>
+            <li>Git basics</li>
+            <li>Networking basics</li>
+            <li>Digital marketing</li>
+            <li>Research data analysis</li>
+            <li>Excel and PowerPoint proficient user</li>
+        </ul>
+    </section>
+    <hr>
+    <section>
+        <h2>Work experience</h2>
+        <p class="paragraph dates"><em>2013 - until present</em></p>
+        <h3>Strategic Planning Director at “InMedias”</h3>
+        <ul>
+            <li>Communication with key clients</li>
+            <li>Project management</li>
+            <li>Market research</li>
+            <li>Media strategy development (online and offline media)</li>
+            <li>Supervision of Planning department</li>
+        </ul>
+        <p class="paragraph dates"><em>2007 - 2013</em></p>
+        <h3>Media Director at “Salmon Media”</h3>
+        <ul>
+            <li>Media and communications strategy development</li>
+            <li>Media market analysis</li>
+            <li>Personnel training</li>
+        </ul>
+
+
+    </section>
+    <section>
+        <hr>
+        <h2>Education</h2>
+        <p class="paragraph">
+            2015 Institute of IT & Business Administration, Cisco Network Academy CCNA, certificate
+            <br>
+            2015 Institute of IT & Business Administration, English for Business and Career, certificate
+            <br>
+            1997 - 2002 Belarusian State University, Faculty of Philosophy and Social Sciences, diploma
+        </p>
+    </section>
+    <section>
+        <hr>
+        <h2>English skills</h2>
+        <p class="paragraph">2015 Institute of IT & Business Administration, “English for Business and Career”, level C1
+        </p>
+        <ul>
+            <li>written communication with customers and partners on a daily basis</li>
+            <li>regular voice communication (skype etc.)</li>
+            <li>composing documents and presentations in English</li>
+            <br>
```

Comment:
Try don't use tags `<br />` and `<hr />` everywhere, you can do the same using styles, 
it's hard to support those tags